lib/AvaTaxClient.js

```diff
@@ -84,7 +84,8 @@ export default class AvaTaxClient {
       },
       body: JSON.stringify(payload)
     }).then(res => {
-      if (res.headers._headers['content-type'][0] === 'application/vnd.ms-excel') {
+      var contentType = res.headers._headers['content-type'][0];
+      if (contentType === 'application/vnd.ms-excel' || contentType === 'test/csv') {
```

Comment:
`contentType == 'text/csv'`

Reply:
@qjavalara makes sense--assuming that the header has also been fixed server-side (it was set to `test/csv` early this week when I checked).